src/sentry/grouping/strategies/newstyle.py

```diff
@@ -791,11 +794,12 @@ def get_first_path(exception: SingleException) -> Generator[SingleException, Non
             yield from get_first_path(children[0])
 
     # Traverse the tree recursively from the root exception to get all "top-level exceptions" and sort for consistency.
-    top_level_exceptions = sorted(
-        get_top_level_exceptions(exception_tree[0].exception),
-        key=lambda exception: str(exception.type),
-        reverse=True,
-    )
+    if exception_tree[0].exception:
```

Comment:
Otherwise I get this error in line 799:
```
src/sentry/grouping/strategies/newstyle.py:799: error: Argument 1 to "get_top_level_exceptions" has incompatible type "SingleException | None"; expected "SingleException"  [arg-type]
Found 1 error in 1 file (checked 1 source file)
```